extensions/rt_ignore_dh/rt_ignore_dh.c

```diff
@@ -165,8 +165,13 @@ static int stow_destination_host(struct msg **msg) {
 			CHECK_FCT(fd_msg_avp_add(pi_avp, MSG_BRW_LAST_CHILD, ph_avp));
 			CHECK_FCT(fd_msg_avp_add(pi_avp, MSG_BRW_LAST_CHILD, ps_avp));
 
-			/* remove Destination-Host from message */
-			fd_msg_free((msg_or_avp*)avp);
+			/* set the Destination-Host to the local Diameter Identity */
```

Comment:
OK.  I think that's a bug, but I'm happy that we can apply this change for now and raise an issue to investigate the absence of Destination-Host later.